examples/spells/basicSpells.qnt

```diff
@@ -0,0 +1,88 @@
+// -*- mode: Bluespec; -*-
+/**
+ * This module collects definitions that are ubiquitous.
+ * One day they will become the standard library of Quint.
+ */
+module basicSpells {
+  /**
+   * An annotation for writing preconditions.
+   * @param __cond condition to check
+   * @returns true if and only if __cond evaluates to true
+   */
```

Comment:
When I was writing `builtin.qnt`, I proposed we used a rust-like doc comment format, with triple slashes (`///`) and markdown inside. Currently, that sort of comment is parsed as a special comment and it's contents are assigned as a definition's documentation (that can be converted to markdown and visualized in overlays in VSCode).


I think this other JSDoc like format has some pros and cons against the triple-slash format. My question here is: do we want to introduce a new documentation format now? Do we want to switch from the one we already have?

I'm ok with changing the format, I'd just like to be consistent here (i.e. this file vs. `builtin.qnt`).

Reply:
ah, that's what I was looking for. Yes, let's use `///`. Would it work with `@param` etc.?

Reply:
I have changed the formatting to `///`. Afaik, it does not work in front of a `module`, right?

Reply:
> ah, that's what I was looking for. Yes, let's use `///`. Would it work with `@param` etc.?

Not really, it assumes the input is simple markdown. We can have some special rules there if we want.

Reply:
> I have changed the formatting to `///`. Afaik, it does not work in front of a `module`, right?

Not yet :disappointed: